### PR TITLE
Wazo 2817 - py3

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 *.pyc
+.coverage
+coverage.xml
+nosetests.xml
```
Comment (fblackburn1) [inline .gitignore:4]: do not hesitate to use pytest instead of nosetests :)

File: tests/test_xivo_update_odbcinst_config.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Copyright 2015-2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0+
```
Comment (bloom1) [inline tests/test_xivo_update_odbcinst_config.py:2]: FYI: the SPDX-License-Identifier is now [GPL-3.0-or-later](https://spdx.org/licenses/GPL-3.0-or-later.html); `GPL-3.0+` has been [deprecated](https://spdx.org/licenses/GPL-3.0+.html)

Comment (sopelj) [reply tests/test_xivo_update_odbcinst_config.py:2]: Ok, sorry, I had just copied the other test. Should I replace the others too? Or not touch them?

**Review by pc-m — COMMENTED** [review]
Good job Jesse. I left a question which I think is a copy/paste error that slipped in.

File: tox.ini [hunk 3]
```diff
@@ -16,21 +16,30 @@ deps =
...
-commands =
-    -sh -c 'pylint --rcfile=/usr/share/xivo-ci/pylintrc sbin/xivo-create-config > pylint.txt'
+[flake8]
+exclude =
+    .tox
+    .eggs
+    plugins/_build
+show-source = true
+max-line-length = 99
+application-import-names = wazo_agid
```
[the remark this answers is not in the document]
Comment (sopelj) [reply tox.ini]: Oops! Good catch. Fixed

**Review by pc-m — APPROVED** [review]

File: tests/test_xivo_update_odbcinst_config.py (hunk 2 above)
Comment (fblackburn1) [reply tests/test_xivo_update_odbcinst_config.py]: Feel free to fix them all (in any repo). Just don't touch the copyright date for this change

Comment (sopelj) [reply tests/test_xivo_update_odbcinst_config.py]: Ok, thanks! I'll keep an eye out.

**Pull request merged**